tidb-cloud/set-up-private-endpoint-connections-on-azure.md

```diff
@@ -0,0 +1,115 @@
+4. [Connect to your TiDB cluster](#step-4-connect-to-your-tidb-cluster)
+
+If you have multiple clusters, you need to repeat these steps for each cluster that you want to connect to using Azure Private Link.
+
+### Step 1. Select a TiDB cluster
+
+1. On the [**Clusters**](https://tidbcloud.com/console/clusters) page, click the name of your target TiDB cluster to go to its overview page.
+2. Click **Connect** in the upper-right corner. A connection dialog is displayed.
+3. In the **Connection Type** drop-down list, select **Private Endpoint**, and then click **Create Private Endpoint Connection**].
+
+> **Note:**
+>
+> If you have already created a private endpoint connection, the active endpoint will appear in the connection dialog. To create additional private endpoint connections, navigate to the **Networking** page in the left navigation pane.
+
+### Step 2. Create Azure private endpoint
+
+1. In **Create Azure Private Endpoint Connection** page, copy the value provided for **Resource ID** of private link service. Do not close the window for later use.
+
+> **Note:**
+>
+> For each TiDB Cloud Dedicated cluster, the corresponding endpoint service is automatically created 3 to 4 minutes after the cluster creation.
+
+2. Log in Azure Console and create a new private endpoint for your cluster using the copied Resource ID. For details, please refer to [Create a private endpoint](https://learn.microsoft.com/en-us/azure/private-link/create-private-endpoint-portal?tabs=dynamic-ip) in Azure documentation.
+    1. Set the **connection method** to "Connect to an Azure resource by resource ID or alias" in **Resource** step.
+    2. Paste the copied Resource ID to the field of "**Resource ID or alias**".
+    ![Create Azure private endpoint using service resource id](/media/tidb-cloud/azure-create-private-endpoint-service-resource-id.png)
+    3. After the private endpoint is created, please go to Settings and copy the following information,    
+        1. Click **Properties** and copy its **Resource ID**. 
+        ![Azure private endpoint resource id](/media/tidb-cloud/azure-private-endpoint-resource-id.png)
+        2. Click **DNS configuration** and copy its **IP address**.
+        ![Azure private endpoint dns ip](/media/tidb-cloud/azure-private-endpoint-dns-ip.png)
+  
+
+### Step 3. Accept endpoint
+
+1. Return to the TiDB Cloud console, and paste the copied **Resource ID** and **IP address** to the corresponding field.
+2. Click **Verify Endpoint** to validate the private endpoint's information. If you meet the error, please check the information and try it again.
+3. Once verification passes, click **Accept Endpoint** to approve the connection from your private endpoint.
+
+### Step 4. Connect to your TiDB cluster
+
+After you have accepted the endpoint connection, you are redirected back to the connection dialog.
+
+1. Wait for the private endpoint connection status to become Active (approximately 5 minutes).
```

Comment:
Does the user need to go to the Networking page to check if the private endpoint connection is active?